### PR TITLE
fix "Failed to list *v1beta1.VolumeSnapshotClass" on k8s 1.16

### PR DESCRIPTION
**Pull request by min-zh**
**What type of PR is this?**

 /kind bug

**What this PR does / why we need it**:
fix bug "Failed to list *v1beta1.VolumeSnapshotClass" on k8s 1.16

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/controller/storage/capability/capability_controller.go [hunk 1]
```diff
@@ -356,7 +364,7 @@ func (c *StorageCapabilityController) deleteStorageCapability(name string) error
 }
 
 func (c *StorageCapabilityController) deleteSnapshotClass(name string) error {
-	if !c.snapshotSupported {
+	if !c.snapshotSupported || c.snapshotClassClient == nil {
```
Comment (zheng1) [inline pkg/controller/storage/capability/capability_controller.go:367]: This line can be refine to an common func

Comment (min-zh) [reply pkg/controller/storage/capability/capability_controller.go:367]: OK

Comment (min-zh) [reply pkg/controller/storage/capability/capability_controller.go]: We already have unit test ‘TestCreateStorageClassNotSupportSnapshot ’ for this case.

**Pull request merged**